### DISCUSSION TITLE
Deployment scripts/guide?

### DISCUSSION THREAD

**Discussion by Gildedter** [firtadokei/ci-svelte-appstarter#2] [Q&A]
Is there a deployment script? I know CI just needs to be copied over after updating some stuff, but I'm unfamiliar with the svelte side of things. I'm running my dev server using laragon (with nginx virtual host).

Edit: nvm running `npm run build` compiles every client-side asset

**Comment by jaguarch** [comment]
didn't get what you mean exactly, but if you are asking about building for production, simply just run `npm run build`, this command will generate production-ready assets in your public directory
